### DISCUSSION TITLE
Complex ordering with Prisma

### DISCUSSION THREAD

**Discussion by alan345** [prisma/prisma#3771] [Q&A] [answered]
Can we do some complex ordering with Prisma? Or I presume I should use raw SQL?

```sql
select *
from table t
where col like '%Ami%'
order by (col = 'Ami') desc, length(col);
```
source: https://stackoverflow.com/questions/31315669/how-to-use-mysql-like-with-order-by-exact-match-first

**Comment by ryands17** [comment, marked as answer]
Hey @alan345 👋 
Currently, this type of ordering isn't supported by Prisma, so you would need to stick to a [raw query](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/raw-database-access). Also, it would be great if you could open a request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look further into it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
